src/components/Layouts/PageLayouts.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC, PropsWithChildren } from "react";
+import { ThemeProvider, Theme, Box } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+import { customTheme } from "../../themes/theme";
+import { Header } from "@forkfacts/components";
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    backgroundColor: theme.palette.background.default,
+  },
+  main: {
+    backgroundColor: theme.palette.background.default,
+  },
+}));
+
+const LayoutComponent: FC<PropsWithChildren> = ({ children }) => {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.root}>
+      <Box className={classes.main}>{children}</Box>
+    </Box>
+  );
+};
+
+const Layout: FC<PropsWithChildren> = ({ children }) => {
```

Comment:
You have just 1 Layout, so why your component name is `PageLayouts`? Rename this to be either `Layout` or `PageLayout`. Be careful when using singular vs plural naming conventions.
Rename the corresponding story to match the correct name.